index.html

```diff
@@ -0,0 +1,542 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Design Teardown Project</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link href="css/style.css" rel="stylesheet" />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Cairo:wght@200;600;900&family=Cormorant+Upright:wght@300;400;500&family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Nunito:wght@200;400;700;800;900&family=Playfair+Display+SC:ital@0;1&family=Roboto:wght@100;300;400&display=swap"
+            rel="stylesheet"
+        />
+        <script src="https://kit.fontawesome.com/cbdf2b0f0e.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header class="top-header flex-row">
+            <div class="header-logo">
+                <a href="#">
+                    <img src="https://d33wubrfki0l68.cloudfront.net/81fa290218cedc39d32be7b1f1aba58a25829ebe/0a05f/images/logo--full.svg" class="header-picture"  alt="y"/>
+                </a>
+            </div>
+            <nav class="navbar">
+                <ul class="flex-row font">
+                    <li class="header-list"><a href="#">Articles</a></li>
+                    <li class="header-list"><a href="#">Guides</a></li>
+                    <li class="header-list"><a href="#">Books</a></li>
+                    <li class="header-list"><a href="#">Events</a></li>
+                    <li class="header-list"><a href="#">Membership</a></li>
+                    <li class="header-list hide"><a href="#11">Jobs</a></li>
+                    <li class="header-list hide"><a href="#211">Newsletter</a></li>
+                    <li class="header-list hide"><a href="#311">Podcast</a></li>
+                    <li class="header-list hide4"><a href="#411">Write for us</a></li>
+                    <li class="header-list hide5"><a href="#511">Advertise with us</a></li>
+                    <li class="header-list flex-row">
+                    <button type="button" class="header-button"><span>More</span><span class="header-cat"></span></button>
+                    </li>
+                </ul>
+            </nav>
+            <div class="search-field">
+                <form class="form-field flex-row">
+                    <button type="button">
+                        <i class="fas fa-search"></i>
+                    </button>
+                    <input type="text" placeholder="Search articles..." class="search-text" />
+                </form>
+            </div>
+        </header>
+        <main class="latest-article">
+            <div class="top-article top1">
```

Comment:
please use for each article an article tag instead of a div, that will be more semantic for your project it's also on the requirements